Build/Scripts/GuardingV2ExtensionsHelper.psm1

```diff
@@ -0,0 +1,229 @@
+<#
+.Synopsis
+    Configure breaking changes check
+.Description
+    Configure breaking changes check by:
+    - Restoring the baseline package and placing it in the app symbols folder
+    - Generating an AppSourceCop.json with version and name of the extension
+.Parameter AppSymbolsFolder
+    Local AppSymbols folder
+.Parameter AppProjectFolder
+    Local AppProject folder
+.Parameter BuildMode
+    Build mode
+#>
+function Enable-BreakingChangesCheck {
+    Param(
+        [Parameter(Mandatory = $true)] 
+        [string] $AppSymbolsFolder,
+        [Parameter(Mandatory = $true)] 
+        [string] $AppProjectFolder,
+        [Parameter(Mandatory = $true)] 
+        [string] $BuildMode
+    )
+
+    # Get name of the app from app.json
+    $appJson = Join-Path $AppProjectFolder "app.json"
+    $applicationName = (Get-Content -Path $appJson | ConvertFrom-Json).Name
+    
+    # Get the baseline version
+    $baselineVersion = Get-BaselineVersion -BuildMode $BuildMode
+
+    Write-Host "Restoring baselines for $applicationName from $baselineVersion"
```

Comment:
Here `$baselineVersion` can be either a proper version from BC artifacts or a version coming from this repository (the latest available one).

The logging might be confusing, as it might appear that for _Clean_ mode we use 21.6.901 and for _Default_ mode we use 21.6.50123 (numbers are made up).

What might be less confusing is to log something like:
_"Enabling breaking changes check in Clean mode, setting up latest X.x version as a baseline"_ 
or
_"Enabling breaking changes check in Default mode, setting up X.x.x.x version as a baseline"_

Reply:
Good point :) Added some logging of this now